### DISCUSSION TITLE
Run a single acceptance test multiple times

### DISCUSSION THREAD

**Discussion by hemantk-12** [apache/ozone#5547] [General]
Hi All, Is there a way to run a single acceptance test multiple times similar to integration tests.
I see there is script [test-single](https://github.com/apache/ozone/blob/5249f99e41939275399fd864a30b09492934fa9f/hadoop-ozone/dist/src/main/compose/test-single.sh) to run single test and probably update it in [acceptance](https://github.com/apache/ozone/blob/754a6bb8264fcfc98d9c9a39cb13d0492ee0e587/hadoop-ozone/dev-support/checks/acceptance.sh#L42) here instead of test-all. I tired but I guess I am not passing appropriate parameters and got error: https://github.com/hemantk-12/ozone/actions/runs/6740551750/job/18324221554

It would be great if I can get some help on this.

**Comment by adoroszlai** [comment]
Currently this is possible only by manually editing the CI workflow (disabling other checks, limiting the suite of tests, etc.)

Prompted by your question, here's a proposed new workflow for making it easier:
https://github.com/apache/ozone/pull/5548

**Comment by hemantk-12** [comment]
Thanks @adoroszlai for adding the new workflow.

This is a manual workflow that can be run on demand for a specified test suite in one's own fork.
* Specify the branch on which the test suite has to run. 
* `Test Suite`, which should be ones of the existing suites from regular CI,  e.g. "cert-rotation", or
* `Test Filter`, which is a regex pattern applied to filter test script's path e.g. "ozone-csi", "test-vault.sh", "ozone/test-ec.sh", "test-.*-rotation.sh".

<img width="1730" alt="Screenshot 2023-11-07 at 10 58 21 AM" src="https://github.com/apache/ozone/assets/6820020/2dcb309c-934e-4bdc-b0e9-d8a404ac8b1a">
